examples/sveltekit/README.md

```diff
@@ -1,30 +1,38 @@
-# SvelteKit
+# create-svelte
 
-Everything you need to build a Svelte project, powered by [`create-svelte`](https://github.com/sveltejs/kit/tree/master/packages/create-svelte), deployed to [Vercel](https://vercel.com).
+Everything you need to build a Svelte project, powered by [`create-svelte`](https://github.com/sveltejs/kit/tree/master/packages/create-svelte).
 
-## Deploy Your Own
+## Creating a project
 
-[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fvercel%2Fvercel%2Ftree%2Fmain%2Fexamples%2Fsveltekit&project-name=sveltekit-vercel&repository-name=sveltekit-vercel&demo-title=SvelteKit%20%2B%20Vercel&demo-description=A%20SvelteKit%20app%20optimized%20Edge-first.&demo-url=https%3A%2F%2Fsveltekit-template.vercel.app%2F)
+If you're seeing this, you've probably already done this step. Congrats!
 
-_Live Example: https://sveltekit-template.vercel.app_
+```bash
+# create a new project in the current directory
+npm create svelte@latest
+
+# create a new project in my-app
+npm create svelte@latest my-app
+```
 
 ## Developing
 
-Once you've installed dependencies with `pnpm install`, start a development server:
+Once you've created a project and installed dependencies with `npm install` (or `pnpm install` or `yarn`), start a development server:
 
 ```bash
-pnpm run dev
+npm run dev
 
 # or start the server and open the app in a new browser tab
-pnpm run dev -- --open
+npm run dev -- --open
 ```
 
 ## Building
 
 To create a production version of your app:
 
 ```bash
-pnpm run build
+npm run build
```

Comment:
Should we keep using pnpm? 🙏 That's our preference.

Reply:
I would prefer that as well but keep in mind this is going to get overwritten every time the cron job from the other PR runs 😬

Reply:
Probably best to leave it -- this README is the one generated by the `create-svelte` CLI, so anything we do to it is going to get overridden, and we can't guarantee it's not going to just change on us whenever.